zuul.d/jobs.yaml

```diff
@@ -1099,3 +1099,24 @@
     override-checkout: branch-2.4
     vars:
       k8s_log_dir: '{{ ansible_user_dir }}/workspace/logs/kubernetes'
+
+- job:
+    name: helm-integration-test-kubeadm-k8s-v1.13.4
+    parent: init-test
+    description: |
+      Run integration tests of helm charts with v1.13.4 k8s cluster deployed by kubeadm
```

Comment:
Which version of helm we use in testing, `latest` or specific version?, please add it into description.